languages/go/exercises/concept/time/.docs/introduction.md

```diff
@@ -0,0 +1,22 @@
+## Time
+
+In Go, functionality for working with times is provided by the [time package][time_pkg]. The types and methods in this package allow us to manipulate times, get the current time, determine elapsed time, parse times from strings, and more.
+
+To work with time, you will usually just call a method on the type you are working with, but there are also some methods called on the package itself. So if we had a `Time`, say, 6:53:34, stored in a variable called `myTimeVar`, and wanted to extract the minute, we could browse through the `time` package and find the `Minute` method:
+
+```go
+    myTimeVar.Minute()
+    // 53
+```
```

Comment:
In `introduction.md` documents, we [only use code examples when new _syntax_ is introduced](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsintroductionmd). This code example is just a regular method invocation, which syntax has been introduced in an earlier exercise. As such, could you remove this example? This means that the text could maybe become something like this:

```suggestion
To work with time, you will usually call a method on a `Time` instance, but there are also some methods called on the `time` package itself.
```